src/main/java/uk/gov/hmcts/reform/em/npa/redaction/PdfRedaction.java

```diff
@@ -80,29 +80,24 @@ private Rectangle createRectangle(
             RectangleDTO rectangle) {
 
         PDPage page = document.getPage(pageNumber);
-        PDRectangle pageSize = page.getMediaBox();
+        PDRectangle pageSize = page.getCropBox();
+
+        float x = pixelToPointConversion(rectangle.getX());
+        float y = pixelToPointConversion(rectangle.getY());
+        float width = pixelToPointConversion(rectangle.getWidth());
+        float height = pixelToPointConversion(rectangle.getHeight());
+
+        // Adjust for top-left origin
+        float adjustedY = pageSize.getUpperRightY() - y - height;
```

Comment:
The only real difference to previous code is using `getUpperRightY()` rather than `getHeight()` - `getHeight()` may not always count margin/offsets, which in turn causes the later calculations to be off, but `getUpperRightY()` should include the whole page.